modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -21,6 +22,7 @@ var (
 	Code                   = []byte("01234567012345670123456701234567")
 	contractClientState    = []byte{1}
 	contractConsensusState = []byte{2}
+	ErrMockContract        = errors.New("mock contract error")
```

Comment:
this might be helpful for other tests that require some error be returnred from contract

Reply:
pulled into separate PR to reduce diff https://github.com/cosmos/ibc-go/pull/4955